client/dialogs/connected-dialog-overlay.jsx

```diff
@@ -27,39 +29,39 @@ class ConnectedDialogOverlay extends React.Component {
     this._focusable = elem
   }
 
-  getDialogComponent(dialogType) {
+  getDialog(dialogType) {
     switch (dialogType) {
       case 'acceptMatch':
-        return AcceptMatch
+        return { component: AcceptMatch, modal: true }
```

Comment:
A little sad we lost the ability to specify modality on the dialog component itself. I feel like there's still a reasonable way to do that with render props or something, but oh well, not worth trying to fix as part of this PR. We can refactor it later.